app/models/review.rb

```diff
@@ -0,0 +1,5 @@
+class Review < ActiveRecord::Base
+  belongs_to :pickup
+  has_one :pickup_reason
+  has_one :warehouse
```

Comment:
we need to have the foreign key in this class, right?
looks like this should be belongs_to instead of has_one